tox.ini

```diff
@@ -1,33 +1,19 @@
 [tox]
 envlist =
-    py27,
-    py33,
-    py34,
-    py35,
-    py36,
-    pypy,
-    pypy3,
-    coverage-report,
+    py27,pypy,py34,py35,py36,pypy3,coverage
 
 [testenv]
 commands =
-    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
+    zope-testrunner --test-path=src []
 deps =
     .[test]
-    zope.testrunner
-    coverage
-setenv =
-    COVERAGE_FILE=.coverage.{envname}
 
-[testenv:coverage-report]
-basepython = python2.7
-deps = coverage
-setenv =
-    COVERAGE_FILE=.coverage
-skip_install = true
+
+[testenv:coverage]
```

Comment:
Counterexample: [This PR in zope.deprecation](https://github.com/zopefoundation/zope.deprecation/pull/11/files) runs always and only with coverage.

Counterargument: [This article](https://blog.ionelmc.ro/2014/05/25/python-packaging/#tl-dr) @tseaver pointed my to stresses that all tests should be run both with *and* without coverage (to detect race conditions, I think).